wpilibj/src/main/native/cpp/InterruptJNI.cpp

```diff
@@ -73,7 +73,11 @@ void InterruptJNI::SetFunc(JNIEnv* env, jobject func, jmethodID mid,
   if (thr->m_param) env->DeleteGlobalRef(thr->m_param);
   // create global references
   thr->m_func = env->NewGlobalRef(func);
-  thr->m_param = param ? env->NewGlobalRef(param) : nullptr;
+  if (param) {
+    thr->m_param = env->NewGlobalRef(param);
+  } else {
+    thr->m_param = nullptr;
+  }
```

Comment:
I guess I'm okay with this, but this is a pretty typical idiom.

Reply:
If it's a common idiom, I guess it's fine.